auth/user_mgt.go

```diff
@@ -173,6 +176,11 @@ func (u *UserToUpdate) PhoneNumber(phone string) *UserToUpdate { u.set("phoneNum
 // PhotoURL setter.
 func (u *UserToUpdate) PhotoURL(url string) *UserToUpdate { u.set("photoUrl", url); return u }
 
+func (u *UserToUpdate) revokeRefreshToken() *UserToUpdate {
```

Comment:
Should this be exposed as `TokensValidAfterTime(since int64)`?

Reply:
I've opened it up for discussion (whether we want the devs to do something like revoke 5 minutes from now, or revoke the old ones before last Tuesday.)